.travis.yml

```diff
@@ -13,17 +13,25 @@ matrix:
   include:
     - compiler: gcc
       env:      MAINT_EXTRA=1
+      arch:     amd64
     - compiler: clang
       env:      MAINT_EXTRA=0
+      arch:     amd64
+    - compiler: gcc
+      env:      MAINT_EXTRA=1
+      arch:     ppc64le 
```

Comment:
MAINT_EXTRA=1 just runs some arch-independent schema tests, so we only need to do it once ... we can drop this and just keep the MAINT_EXTRA=0 part.

Reply:
Thanks !! As per your suggestion, i have updated it.